src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsFeed.java

```diff
@@ -221,8 +224,9 @@ private String getSaveFilePath(String gtfsFeedUrl, HttpURLConnection connection)
         } else {
             //Extracts file name from header field
             int index = disposition.indexOf("filename=");
+            int filenameLastIndex = disposition.indexOf("\"", index+10);
```

Comment:
I believe this will fail on Linux and Mac due to hard coded Windows path character \. Could you please see if you can make this platform independent?